src/lib/utils.test.ts

```diff
@@ -86,6 +112,25 @@ describe('toaGetRunsRequest', () => {
         process.env.TOA_BASIC_AUTH = ''
         await expect(toaGetRunsRequest()).rejects.toThrow('TOA token failed to generate')
     })
+
+    it('should throw an error if response status is not success', async () => {
+        global.fetch = vi.fn().mockResolvedValue(new Response('Authentication error', { status: 401 }))
+        await expect(toaGetRunsRequest()).rejects.toThrow(
+            'Received an unexpected 401 from trusted output app: Authentication error',
+        )
+    })
+
+    it('should throw an error if response structure is unexpected', async () => {
+        const mockTOAData = {
+            // Run has unexpcted type for runId
```

Comment:
Little typo

Reply:
Thanks for the catch / extra :eye:s! -- Fixed [here](https://github.com/safeinsights/setup-app/pull/11/commits/7bd2ac7cc909a931d7372823a144d952d0b222b8)